lib/bson.rb

```diff
@@ -38,22 +39,22 @@ def self.ObjectId(string)
   # Constant for binary string encoding.
   #
   # @since 2.0.0
-  BINARY = "BINARY".freeze
+  BINARY = "BINARY"
 
   # Constant for bson types that don't actually serialize a value.
   #
   # @since 2.0.0
-  NO_VALUE = "".force_encoding(BINARY).freeze
+  NO_VALUE = -("".b)
```

Comment:
Are there any performance benefits in replacing `"".force_encoding(BINARY)` with `-("".b)`? The former is way more readable to my taste.

Reply:
yes, it's faster, but it really doesn't matter for constants.

since the gem does support Ruby 2.5+, this ```String.new(encoding: BINARY)``` should be even better, what do you think?

the previous ```"".force_encoding(BINARY)``` won't work anymore, because "" is now frozen and force_encoding is a destructive operation